public/data/7flash.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Berlian Gur",
+  "type": "personal",
+  "bio": "Software Engineer",
+  "avatar": "https://avatars.githubusercontent.com/u/4569866?v=4",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/7flash",
+      "icon": "github"
+    }
+  ],
+  "milestones": [
+        "title": "Joined Github",
```

Comment:
I recommend capitalizing the `h` here. What do you think?
```suggestion
        "title": "Joined GitHub",
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question